.gitignore

```diff
@@ -3,3 +3,4 @@
 .pub/
 packages
 pubspec.lock
+build/
```

Comment:
I don't know why we'd have a build directory in this repo... Is this an artifact of a local tool you are using? Consider adding this to a global ignore if so.

Reply:
Hmm, when I ran the unit tests, they seemed to generate a build folder, so I added to `.gitignore`. I think I ran them within VS Code, so maybe that's why. 

I'm happy with removing that change, though.